docs/pages/install.mdx

```diff
@@ -2,44 +2,35 @@ import DownloadsComponent from "/components/DownloadsComponent";
 
 # Install Pelican
 
-This document lists the operating system requirements to install Pelican and explains how a user can download Pelican client or Pelican server components.
+This document lists the operating system requirements for Pelican and explains how a user can download and install Pelican.
+
+## Before Starting
+
+Pelican executables can run as a **client** or a **server**.
+
+For client usage, such as accessing the data, you can download and install a Pelican package or Pelican binary [following the instructions below](#install-pelican-binary).
+
+For server usage, such as serving your data to a Pelican data federation or start a new Pelican data federation, you need to run Pelican using [Pelican Docker images](./install/docker.mdx). It is not recommended to run Pelican server with a binary. Although possible, additional software dependencies and configurations are required to allow the server to be functional.
```

Comment:
Small typo -- "... to a Pelican data federation or _to_ start a new Pelican ..."